rust/kernel/revocable.rs

```diff
@@ -127,29 +167,20 @@ impl<T: ?Sized> Drop for Revocable<T> {
 /// The RCU read-side lock is held while the guard is alive.
 pub struct RevocableGuard<'a, T: ?Sized> {
     data_ref: *const T,
+    _rcu_guard: rcu::Guard,
     _p: PhantomData<&'a ()>,
 }
```

Comment:
This changes `RevocableGuard` to be non-Send, which I suspect was intended to be the case in the first place.

Reply:
Yes, it was meant to be non-Send in the first place.